components/resources/brave_components_strings.grd

```diff
@@ -814,6 +814,10 @@
       <message name="IDS_BRAVE_WALLET_DOWNLOADING" desc="">Downloading Component...</message>
       <message name="IDS_BRAVE_WALLET_DISCLOSURE_CONFIRM" desc="">I understand</message>
 
+      <message name="IDS_PAGE_INFO_BRAVE_WEBTORRENT" desc="The message showed to the user in the HTTP info bubble view when viewing a webtorrent: page">
+        You're viewing a WebTorrent page
```

Comment:
Maybe this should mention if this page is "secure"? cc: @fmarier

Reply:
The page itself is always secure (it has the `chrome-extension://` scheme). I suppose you were asking about differentiating between an HTTP torrent v. an HTTPS torrent or a magnet link?

Reply:
If so, I think that would be an interesting distinction to make in the WebTorrent UI. It would be the equivalent of the "Not secure" warning we see for `http://` pages but for torrents. That sounds like it should be done independently of this fix though since it's an existing issue.